doc/theory/glossary.tex

```diff
@@ -16,56 +68,254 @@
+	
+	\subsection*{Metric space:}
+	A  pair $(X, d)$ where $X$ is a set and $d$ is a function 
+	$$
+	d : X \times X \to \mathbb R
+	$$
+	attaining non-negative values is called a \textit{metric space} if
+	$$d(x,y) = 0\ \Leftrightarrow\  x = y$$
+	$$d(x,y) = d(y,x)$$
+	$$d(x,z) \leq d(x,y) + d(y, z)$$
 	
 	In this case, the function $d$ is refer to as the \textit{metric} and the value $d(x,y)$ is called the \textit{distance} between $x$ and $y$. 
 	
-	\paragraph{Point cloud:} A \textit{point cloud} is a finite metric space. A common type of point cloud is defined by a finite subset of $\mathbb R^n$ with metric induced from the eucliden distance.
+	\subsection*{Euclidean distance and norm:}
+	The set $\mathbb R^n$ defines a metric space with euclidean distance
+	\begin{equation*}
+	d(x,y) = \sqrt{(x_1-y_1)^2 + \cdots + (x_n-y_n)^2}.
+	\end{equation*}
+	
+	The norm $||x||$ of a vector $x$ is defined as its distance to the $0$ vector.
+	
+	\subsection*{Finite metric spaces and point clouds:} A \textit{finite metric space} is a finite set together with a \underline{metric}. A \textit{distance matrix} associated to a finite metric space is obtained by choosing a total order on the finite set and setting the $(i,j)$-entry to be equal to the distance between the $i$-th and $j$-th elements. 
+	
+	A \textit{point cloud} is a finite subset of $\mathbb{R}^n$ (for some $n$) together with the metric induced from the \underline{eucliden distance}.
+	
+	\section{Time series}
+	
+	\subsection*{Time series:}
+	A \textit{time series} is a sequence $\{y_i\}_{i = 0}^n$ of real numbers. 
+	
+	A common construction of a times series $\{x_i\}_{i = 0}^n$ is given by choosing $x_0$ arbitrarily as well as a step parameter $h$ and setting
+	\begin{equation*}
+	x_i = x_0 + h\cdot i.
+	\end{equation*}
+	Another usual construction is as follows: given a time series $\{x_i\}_{i = 0}^n \subseteq U$ and a  function
+	\begin{equation*}
+	f :  U \subseteq \mathbb R \to \mathbb R
+	\end{equation*}
+	we obtain a new time series $\{f(x_i)\}_{i = 0.}^n$
+	
+	Generalizing the previous construction we can define a time series from a function 
+	\begin{equation*}
+	\varphi : U  \times M \to M, \qquad U \subseteq \mathbb R, \qquad M \subseteq \mathbb R^d
+	\end{equation*} 
+	using a function $f : M \to \mathbb R$ as follows: let $\{t_i\}_{i=0}^n$ be a time series taking values in $U$, then
+	\begin{equation*}
+	\{f(\varphi(t_i, m))\}_{i=0}^n.
+	\end{equation*}
+	for an arbitrarily chosen $m \in M$.
+	
+	\subsection*{Takens embedding:}	
+	Let $M \subset \mathbb R^d$ be a \underline{compact manifold} of dimension $n$. Let
+	\begin{equation*}
+	\varphi : \mathbb R  \times M \to M
+	\end{equation*} 
+	and
+	\begin{equation*}
+	f : M \to \mathbb R
+	\end{equation*}
+	be generic smooth functions. Then, for any $\tau > 0$ the map
+	\begin{equation*}
+	M \to \mathbb R^{2n+1}
+	\end{equation*}
+	defined by
+	\begin{equation*}
+	x \mapsto\big( f(x), f(x_1), f(x_2), \dots, f(x_{2n}) \big)
```

Comment:
If. time-series people come here, they might be confused by this.
Is it possible to have an example, stating to what \phi and f correspond?